build/env.js

```diff
@@ -0,0 +1,7 @@
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
```

Comment:
Nit: Not sure if we can use `import.meta.dirname` given it's still experimental in Node.js 22.

```suggestion

const __dirname = import.meta.dirname;
```

Reply:
Technically, we require `>=20.0.0` in our `package.json`, so ESLint might complain about us using an API that was marked as stable only after that.

If ESLint is properly checking the `eslint-plugin-n` rules in this repo, then it should give us a `no-unsupported-features/node-builtins` error if we try and use it. However, in that case, a solution would be to bump our Node requirement to `>=22.16.0`.